src/main/java/com/btk5h/skriptmirror/skript/CondAcceptsChange.java

```diff
@@ -0,0 +1,123 @@
+package com.btk5h.skriptmirror.skript;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.UnparsedLiteral;
+import ch.njol.skript.util.Patterns;
+import ch.njol.skript.util.Utils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+
+public class CondAcceptsChange extends Condition {
+
+  private static final Patterns<ChangeMode> PATTERNS = new Patterns<>(new Object[][] {
+      {"%classinfo% can be added to %expressions%", ChangeMode.ADD},
+      {"%classinfo% (can't|cannot) be added to %expressions%", ChangeMode.ADD},
+      {"%expressions% can be set to %classinfo%", ChangeMode.SET},
+      {"%expressions% (can't|cannot) be set to %classinfo%", ChangeMode.SET},
+      {"%classinfo% can be removed from %expressions%", ChangeMode.REMOVE},
+      {"%classinfo% (can't|cannot) be removed from %expressions%", ChangeMode.REMOVE},
+      {"all %classinfo% can be removed from %expressions%", ChangeMode.REMOVE_ALL},
+      {"all %classinfo% (can't|cannot) be removed from %expressions%", ChangeMode.REMOVE_ALL},
+      {"%expressions% can be deleted", ChangeMode.DELETE},
+      {"%expressions% (can't|cannot) be deleted", ChangeMode.DELETE},
+      {"%expressions% can be reset", ChangeMode.RESET},
+      {"%expressions% (can't|cannot) be reset", ChangeMode.RESET}
```

Comment:
Would it be worth supporting multiple classinfos, do you think?
```
potion effects and items can be removed from player
```

Reply:
i can give it a try. the problem is how detecting the plurality of the classinfo is done. i think it might be possible now that i have a better handle on the process.